src/components/App/App.component.jsx

```diff
@@ -1,16 +1,22 @@
 import React, { useLayoutEffect } from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
-
+import { createGlobalStyle } from 'styled-components';
 import AuthProvider from '../../providers/Auth';
-import HomePage from '../../pages/Home';
 import LoginPage from '../../pages/Login';
 import NotFound from '../../pages/NotFound';
 import SecretPage from '../../pages/Secret';
 import Private from '../Private';
-import Fortune from '../Fortune';
 import Layout from '../Layout';
+import VideoList from '../VideoList/VideoList';
 import { random } from '../../utils/fns';
 
+const GlobalStyles = createGlobalStyle`
```

Comment:
I would change this into another file to keep it clean, these global styles might grow in the future.